### ISSUE TITLE
Reduced tyrosine in urine

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6046]
**Preferred term label:** Reduced tyrosine in urine

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of tyrosine in the urine is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormality of urine homeostasis HP:0003110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@JillMiller  -- I do not think that reduced tyrosine is a medical phenotype since normal urine has at most trace levels of tyrosine. Can you specify what this refers to in NSTRN?

**Comment by JillMiller** [comment]
@pnrobinson When I first looked to see if this could be a term I saw ranges for normal values so I thought it could be reduced. If the lower end of those ranges is trace (they likely are), then I agree that this would not be an HPO term.

Here is where I saw that the reference range for tyrosine had a lower end: https://uvmlabs.testcatalog.org/show/AAPD

**Comment by pnrobinson** [comment]
I do not think this is a medical abnormality!
